birdhouse/default.env

```diff
@@ -89,6 +91,16 @@ export MAGPIE_USER_REGISTRATION_NOTIFY_ENABLED=false
 export MAGPIE_LOG_LEVEL=INFO
 export TWITCHER_LOG_LEVEL=INFO
 
+# Endpoint to verify Magpie/Twitcher authorization to a service/resource by a user without proxy request
+# Requires Twitcher>=0.8.0, Required for 'optional-compontents/secure-data-proxy'
+# Uses single quotes for DELAYED_EVAL mechanic to infer the default location based on the expected proxy path.
+export TWITCHER_VERIFY_PATH='$(echo "${TWITCHER_PROTECTED_PATH}" | sed "s/proxy/verify/")'
+
+# When 'optional-compontents/secure-data-proxy' is not enabled, this ensures the variable is defined for nginx.
+# Otherwise, it raises that it is not defined. We want it to be empty when this component is not used.
+# When the component is enabled, the value should be defined by its own 'default.env' file.
+export SECURE_DATA_PROXY_AUTH_INCLUDE=""
```

Comment:
Variable from `optional-compontents/secure-data-proxy` leaking to the main `default.env`?  There is no way to not have this?

Reply:
It must be there to resolve what @matprov flagged in https://github.com/bird-house/birdhouse-deploy/pull/283#discussion_r1099254990. 
This value must be set empty to remove this part: https://github.com/bird-house/birdhouse-deploy/blob/secure-data-proxy/birdhouse/config/proxy/conf.d/all-services.include.template#L42